### PR TITLE
[WIP] tests: Add tests

### PR DESCRIPTION
**Pull request by ayan-b**
Related to https://github.com/moremoban/pypi-mobans/issues/39 & https://github.com/moremoban/pypi-mobans/issues/19

### CODE REVIEW

**Comment by ayan-b** [comment]
What should be our test architecture? We can make dummy repo and then compare or look for specific parts in the rendered file as done here? @jayvdb @chfw

**Comment by chfw** [comment]
I think per each template file, there shall exist a test case for it. @jayvdb , how did your organisation handle it? what's the best practice?

File: .travis.yml [hunk 1]
```diff
@@ -23,3 +23,4 @@ stage: test
 script:
   - find templates/ -type f -name '*.jj2' -exec echo '{}' \;|sed -e "s/templates\//{%include \"/" -e "s/$/\" %}/" > test.file
   - moban -c config/data.yml -t test.file -td templates .
+  - py.test
```
Comment (jayvdb) [inline .travis.yml]: use `pytest`

File: tests/test___init__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import jinja2
+import pytest
+
+path = './templates'
```
Comment (jayvdb) [inline tests/test___init__.py]: use all caps, and a more detailed name

**Comment by ayan-b** [comment]
What have we decided on the test architecture? :smiley:

File: tests/test___init__.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import jinja2
+import pytest
```
Comment (chfw) [inline tests/test___init__.py]: I dont think pytest import is useful here.

**Pull request merged**